### ISSUE TITLE
lacking image prevents epub reading

### ISSUE THREAD

**Issue by fgallaire** [aerkalov/ebooklib#224]
lacking image prevents epub opening

**Comment by aerkalov** [comment]
Yeah. It should not throw exception probably. Would be better if it loads the EPUB and still notices user somehow that some files are missing.

**Comment by fgallaire** [comment]
> Yeah. It should not throw exception probably. Would be better if it loads the EPUB and still notices user somehow that some files are missing.

Yes, as ebook readers are doing

**Comment by fgallaire** [comment]
[T01_-_Noires_visions.zip](https://github.com/aerkalov/ebooklib/files/6019344/T01_-_Noires_visions.zip)

Another example

**Comment by fgallaire** [comment]
[Lune de miel en enfer - Fredric,Brown.zip](https://github.com/aerkalov/ebooklib/files/6026063/Lune.de.miel.en.enfer.-.Fredric.Brown.zip)
[Une etoile m'a dit - Fredric,Brown.zip](https://github.com/aerkalov/ebooklib/files/6026064/Une.etoile.m.a.dit.-.Fredric.Brown.zip)

Same problem for a lacking xhtml file

**Comment by fgallaire** [comment]
[Uploading Avant_Dune,_tome_1_La_Maison_des_Atreïdes_Brian_Herbert_&_Kevin.zip…]()

**Comment by fgallaire** [comment]
will it be fixed ?
